cli/test/e2e/package_authoring_e2e_test.go

```diff
@@ -284,6 +284,9 @@ type promptOutput struct {
 	stringReader io.Reader
 	outputWriter io.Writer
```

Comment:
In theory output reader can be removed all together, I left it in there in case we use prompt reader in another context where we need it. I am open to removing it for now and putting it back together if we think it is needed